templates/selenium3.141.x-python37/src/__ALGO__.py

```diff
@@ -12,20 +13,26 @@ def get_chrome():
     return driver
 
 def get_phantomjs():
-    driver = webdriver.PhantomJS()
+    # PhantomJS must put logs in a place where it has write permissions
+    driver = webdriver.PhantomJS(service_log_path="/tmp/ghostdriver.log")
     return driver
 
 # API calls will begin at the apply() method, with the request body passed as 'input'
 # For more details, see algorithmia.com/developers/algorithm-development/languages
 def apply(input):
+    # note: you MUST have internet access enabled for this test work
     driver = get_chrome()
     #driver = get_phantomjs()
     driver.get("http://www.python.org")
-    assert "Python" in driver.title
+    if 'Python' not in driver.title:
+        return {"status": "failure",
+            "message": "failed to load page"}
     elem = driver.find_element_by_name("q")
     elem.clear()
     elem.send_keys("pycon")
     elem.send_keys(Keys.RETURN)
-    assert "No results found." not in driver.page_source
+    if 'No results found' in driver.page_source:
+        return {"status": "failure",
+            "message": "Failed to find required text in page body"}
     driver.close()
     return {"status": "success"}
```

Comment:
I feel like this would be more useful if this returned the title and body of the page cause right now you aren't really doing much here

Reply:
These checks are based on the official selenium examples.

We wanted to we in sync w/ that as much as possible.